deployment/updates/scripts/add_permissions/2015_10_25_add_like_list_permission.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * @package deployment
+ * 
+ * Add like->list permission-item
+ */
+
+const BASE_USER_ROLE_SYSTEM_NAME = "Basic User Session Role";
+const LIKE_LIST_USER_PERMISSION_TO_ADD = "LIKE_LIST_USER";
+
+$script = realpath(dirname(__FILE__) . '/../../../../') . '/alpha/scripts/utils/permissions/addPermissionsAndItems.php';
+$config = realpath(dirname(__FILE__)) . '/../../../permissions/partner.0.ini';
+passthru("php $script $config");
+
+$config = realpath(dirname(__FILE__)) . '/../../../permissions/service.like.like.ini';
+passthru("php $script $config");
+
+require_once(__DIR__ . "/../../../../" . "alpha/scripts/bootstrap.php");
+
+$c = new Criteria();
```

Comment:
Please passthru to alpha/scripts/utils/permissions/addPermissionToRole.php instead.